### PR TITLE
docs: auto imports don't work in `server/api` 

### PR DESCRIPTION
**Pull request by vencho-mdp**
### 🔗 Linked issue

[composable auto import improvements](https://github.com/nuxt/framework/issues/1273)

### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/content/2.concepts/4.auto-imports.md [hunk 1]
```diff
@@ -56,3 +56,15 @@ Every Nuxt auto-import is exposed by the `#imports` alias that can be used to ma
   const double = computed(() => count.value * 2)
 </script>
 ```
+
+## Imports in `server/api`
```
Comment (danielroe) [inline docs/content/2.concepts/4.auto-imports.md:60]: How about adding this note earlier in the document (just after the line `These functions can be used in components, composables or plugins.`)? https://v3.nuxtjs.org/concepts/auto-imports#nuxt-auto-imports

I'd avoid importing composables in the server API code, personally, so I'm reluctant to use this as an example in the docs.

**Review by pi0 — APPROVED** [review]
LGTM. Thanks!

**Comment by vencho-mdp** [comment]
@danielroe Yes! I completely agree with your idea. However, I also moved that section (as you can see in the PR) a little bit above where it was, because if we leave this under the `Nuxt Auto Imports` header, developers might think that this only applies to those imports.

**Comment by danielroe** [comment]
A good solution!

**Pull request merged**